TestRailAutomationTest/Test/LoginPageTest.cs

```diff
@@ -18,11 +18,15 @@ public void Init()
     }
     
     [Test]
-    public void Test()
+    public void LoginTest()
```

Comment:
https://dzone.com/articles/7-popular-unit-test-naming

Please choose your test naming style and use for all tests